src/serlo-editor-repo/plugin-text/plugins/math.ts

```diff
@@ -4,7 +4,11 @@ export const withMath = (editor: SlateEditor) => {
   const { isInline, isVoid } = editor
 
   editor.isInline = (element) => {
-    return element.type === 'math' ? true : isInline(element)
+    if (element.type !== 'math') {
+      return isInline(element)
+    }
+
+    return element.inline
```

Comment:
just an optinion: I prefer the inline syntax over the is-not logic here.

Reply:
Good suggestion! Changed it.